### DISCUSSION TITLE
How to use it in cloud functions

### DISCUSSION THREAD

**Discussion by jfabregascolvin** [joshcarty/google-searchconsole#57] [Q&A]
Hi, Im using this library in Colab and it´s working very well.

My question is, how can i use it in cloud functions in GCP? I need to declare the library and version in the requirements.txt file but im not sure how to do it.

Another question is how to authenticate searchconsole. In Colab I need to copy a code and paste it by using the created token, but this way is not going to be possible by running the code in a cloud function

Thank you

**Comment by Swchuynh** [comment]
No.!!

**Reply by Swchuynh** [reply]
No.Swc
